tests/cli/workflow/fastq/test_fastq_base.py

```diff
@@ -1,20 +1,20 @@
 import logging
 from datetime import datetime
 
-from cg.cli.workflow.fastq.base import store_available_fastq_analysis, store_fastq_analysis
+from cg.cli.workflow.fastq.base import store_raw_data_analysis, store_available_raw_data_analysis
 from cg.constants.constants import CaseActions, Workflow
 from cg.store.models import Analysis, Case, Sample
 
 
-def test_store_fastq_analysis(another_case_id: str, cli_runner, fastq_context, helpers):
+def test_store_raw_data_analysis(another_case_id: str, cli_runner, fastq_context, helpers):
```

Comment:
Maybe you can also rename the `fastq_context`

Reply:
in a sense it is still a fastq context, as raw-data can be both fastq and now bam. I can specify it being raw-data-fastq